docs/examples.md

```diff
@@ -0,0 +1,99 @@
+Textbook algorithm determining properties of a hidden "oracle" function
+
+*    [Bell Inequality](https://github.com/quantumlib/Cirq/blob/master/examples/bell_inequality.py)
+Demonstration of a Bell inequality which shows violation of local hidden variables
+
+*    [BB84](https://github.com/quantumlib/Cirq/blob/master/examples/bb84.py)
+Textbook algorithm for Quantum key distribution
+
+*    [Noisy simulation](https://github.com/quantumlib/Cirq/blob/master/examples/noisy_simulation_example.py)
+     How to use a noisy simulator to generate results with amplitiude damping.
+
+*    [Line placement](https://github.com/quantumlib/Cirq/blob/master/examples/place_on_bristlecone.py)
+How to find a line of adjacent qubits on a device
+
+*    [Quantum Teleportation](https://github.com/quantumlib/Cirq/blob/master/examples/quantum_teleportation.py)
+A demonstration of using 2 classical bits to transport a quantum state from one
+qubit to another.
+
+*    [Super dense coding](https://github.com/quantumlib/Cirq/blob/master/examples/superdense_coding.py)
+     Transmit 2 classical bits using one quantum bit.
+
+
+# Intermediate Textbook Algorithms
+
+*    [Grover Algorithm](https://github.com/quantumlib/Cirq/blob/master/examples/grover.py)
+Example of applying the textbook Grover algorithm to find a 2-bit oracle
+function.
+
+*    [Quantum Fourier Transform](https://github.com/quantumlib/Cirq/blob/master/examples/quantum_fourier_transform.py)
+A demonstration of a 4-qubit quantum fourier transform (QFT)
+
+*    [Basic Arithmetic](https://github.com/quantumlib/Cirq/blob/master/examples/basic_arithmetic.py)
+Algorithms for adding and multiplying numbers as represented by qubit states
+
+*    [Phase estimation](https://github.com/quantumlib/Cirq/blob/master/examples/phase_estimator.py)
+     Textbook algorithm for phase estimation
+
+*    [Shor](https://github.com/quantumlib/Cirq/blob/master/examples/shor.py)
+Shor's algorithm for factoring.
+
+*    [QAOA](https://github.com/quantumlib/Cirq/blob/master/examples/qaoa.py)
+Demonstration of the quantum approximation optimization algorithm (QAOA) on a
+max-cut problem.
+
+
+## Intermediate NISQ techniques
+
+*    [XEB](https://github.com/quantumlib/Cirq/blob/master/examples/cross_entropy_benchmarking_example.py)
+Shows how to use cross-entropy benchmarking (XEB) to distinguish two
+distributions
+
+*    [Direct fidelity](https://github.com/quantumlib/Cirq/blob/master/examples/direct_fidelity_estimation.py)
+Shows how to use direct fidelity estimation to distinguish a desired state from
+the actual state.
+
+*    [Qubit Characterization](https://github.com/quantumlib/Cirq/blob/master/examples/qubit_characterizations_example.py)
+     How to characterize a qubit using Rabi oscillations, randomized
+     benchmarking, and tomography
+
+
+*    [Swap networks](https://github.com/quantumlib/Cirq/blob/master/examples/swap_networks.py)
+
```

Comment:
Suggestion: "Algorithm for efficiently emulating full connectivity on a limited connectivity grid of qubits."

Reply:
Done.